doc/conf.py

```diff
@@ -110,9 +108,41 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'nature'
+html_theme = "sphinx_rtd_theme"
+
+html_theme_options = {
+    'collapse_navigation': False,
+    'sticky_navigation': True,
+    'logo_only': True,
+}
+
+# List versions
+current_version = os.environ['VERSION']
+html_theme_options.update({'current_version': current_version})
+with open('versions', 'r') as f:
+    _versions = [c[2:] for c in f.read().splitlines()]
+_versions = list(filter(lambda c: not(c.startswith('.')), _versions))
+html_theme_options.update({'versions': [(c, f'../{c}/index.html')
+                                        for c in set(_versions).union([current_version])]})
+
+# Get logo
+path_to_image = "images/tda_logo.svg"
+if not(os.path.exists(path_to_image)):
+    import requests
+    r = requests.get('https://www.giotto.ai/static/vector/logo-tda.svg')
+    with open(path_to_image, 'wb') as f:
+        f.write(r.content)
```

Comment:
I would have just included the svg in the repo (unless you expect it to change often), but OK.

Reply:
indeed, you are right. I will

Reply:
@wreise this is now done, right? Does it mean this code can be eliminated?

Reply:
yes

Reply:
Ok, but maybe let's leave it for the subsequent PR.